docs/competition/teams/process-docs/overnight-battery-loan.md

```diff
@@ -0,0 +1,13 @@
+# Overnight Battery Loan
+
+Teams often request a battery and charger for use overnight to test their robots.
+
+## Obtaining batteries
+
+Batteries and chargers should be collected from helpdesk by teams. Teams should be given a charger, and no more than 2 batteries.
+
+The teams TLA, and the part codes of the batteries and changer should be recorded on the [Overnight battery loan]() form.
```

Comment:
missing link here?

Reply:
Ah, yes. I'll populate it once https://github.com/srobo/kit-coordination-docs/pull/6 merges.

Reply:
@RealOrangeOne can you populate?